### PR TITLE
Add /help and /info as proposed in #26 and #31

### PR DESCRIPTION
**Pull request by TimJentzsch**
`/help` Output:

> Did you find a bug or have any questions?
> Visit the [GitHub](https://github.com/zachkont/dotaUpdatesBot) page for more information, or contact [/u/karaflix](https://www.reddit.com/message/compose/?to=karaflix) on reddit!

`/info` Output:

> The main use of this bot is to notify you whenever a Dota update gets released.
> Visit the [GitHub page](https://github.com/zachkont/dotaUpdatesBot) for more information on what you can do or how to contribute!

### CODE REVIEW

File: updater.py [hunk 1]
```diff
@@ -84,6 +84,14 @@ def notify_users_and_groups(message):
     for user_id in loadjson("userlist"):
         try:
             bot.send_message(user_id, message, parse_mode="Markdown")
+        except telebot.apihelper.ApiException as ex:
```
Comment (zachkont) [inline updater.py:87]: Have you tested this?

File: updater.py [hunk 2]
```diff
@@ -92,6 +100,14 @@ def notify_users_and_groups(message):
     for gid in loadjson("grouplist").keys():
         try:
             bot.send_message(gid, message, parse_mode="Markdown")
+        except telebot.apihelper.ApiException as ex:
+            GROUP_BLOCK_ERROR = "Forbidden: bot was kicked from the group chat"
```
Comment (zachkont) [inline updater.py:104]: Have you tested this?

Comment (TimJentzsch) [reply updater.py:104]: I tested the concept on a seperate bot, unforunitely I can't test it on the actual bot.
Should work though.

Comment (zachkont) [reply updater.py:104]: A separate bot is fine, as long as you tested the exact same scenario. Ok cool, merging this

Comment (TimJentzsch) [reply updater.py:104]: Alright, let me know if anything unexpected happens. Will fix ASAP.

**Pull request merged**